templates/selenium/0/rancher-compose.yml

```diff
@@ -0,0 +1,147 @@
+      label: Selenium Hub Version
+      description: Selenium Hub Version
+      required: true
+      type: enum
+      options:
+        - 3.8.1-bohrium
+        - 3.8.1-aluminum
+        - 3.7.1-cadmium
+        - 3.7.1-beryllium
+        - 3.7.1-argon
+        - 3.7.0-arsenic
+        - 3.6.0-darmstadtium
+        - 3.6.0-copper
+        - 3.6.0-bromine
+        - 3.6.0-americium
+        - 3.5.3-boron
+        - 3.5.3-astatine
+        - 3.5.2-antimony
+        - 3.5.1-aluminum
+        - 3.5.0-argon
+        - 3.4.0-francium
+        - 3.4.0-einsteinium
+        - 3.4.0-dysprosium
+        - 3.4.0-chromium
+        - 3.4.0-bismuth
+    - variable: GRID_TIMEOUT
+      label: Hub Configuration
+      description: Grid Timeout (in seconds)
+      type: int
+      default: 30
+    - variable: PUBLISH_PORT
+      label: Publish Port
+      description: Port to expose for external access
+      type: int
+      default: 4444
+    - variable: DEPLOY_SELENIUM_CHROME
+      description: Deploy Selenium Chrome Node
+      label: Deploy Selenium Chrome Node
+      required: true
+      type: enum
+      default: true
+      options:
+        - true
+        - false
+    - variable: SELENIUM_CHROME_VERSION
+      label: Selenium Chrome Version
+      description: Selenium Chrome Version
+      required: true
+      type: enum
+      options:
+        - 3.8.1-bohrium
+        - 3.8.1-aluminum
+        - 3.7.1-cadmium
+        - 3.7.1-beryllium
+        - 3.7.1-argon
+        - 3.7.0-arsenic
+        - 3.6.0-darmstadtium
+        - 3.6.0-copper
+        - 3.6.0-bromine
+        - 3.6.0-americium
+        - 3.5.3-boron
+        - 3.5.3-astatine
+        - 3.5.2-antimony
+        - 3.5.1-aluminum
+        - 3.5.0-argon
+        - 3.4.0-francium
+        - 3.4.0-einsteinium
+        - 3.4.0-dysprosium
+        - 3.4.0-chromium
+        - 3.4.0-bismuth
+    - variable: SELENIUM_CHROME_SCALE
+      label: Number of chrome container
+      description: Number of chrome container
+      type: int
+      default: 1
+    - variable: DEPLOY_SELENIUM_FIREFOX
+      description: Deploy Selenium Firefox Node
+      label: Deploy Selenium Firefox Node
+      required: true
+      type: enum
+      default: true
+      options:
+        - true
+        - false
+    - variable: SELENIUM_FIREFOX_VERSION
+      label: Selenium Firefox Version
+      description: Selenium Firefox Version
+      required: true
+      type: enum
+      options:
+        - 3.8.1-bohrium
+        - 3.8.1-aluminum
+        - 3.7.1-cadmium
+        - 3.7.1-beryllium
+        - 3.7.1-argon
+        - 3.7.0-arsenic
+        - 3.6.0-darmstadtium
+        - 3.6.0-copper
+        - 3.6.0-bromine
+        - 3.6.0-americium
+        - 3.5.3-boron
+        - 3.5.3-astatine
+        - 3.5.2-antimony
+        - 3.5.1-aluminum
+        - 3.5.0-argon
+        - 3.4.0-francium
+        - 3.4.0-einsteinium
+        - 3.4.0-dysprosium
+        - 3.4.0-chromium
+        - 3.4.0-bismuth
```

Comment:
All version enum variables are same defined, distinct versions between different components could be mixed?? Is mandatory that all versions should be the same?? If yes, i propose you just define one `SELLENIUM_VERSION` enum definition.

Anyway, please define a default value on them.

Reply:
Though it might be useful to have different version of chrome and firefox, will start with single working version and if there is a request will add separate version in future.